libs/crucible/cryptol.rs

```diff
@@ -18,6 +18,11 @@ pub fn override_<F>(f: F, module_path: &str, name: &str) {
     unimplemented!("cryptol::override")
 }
 
+/// Mark the given name to be treated as uninterpreted for the duration
+/// of the current test.
```

Comment:
Please note in the doc comment that this applies only within Cryptol code that you call during the current test - it doesn't affect normal Rust functions.